apps/ryal_core/lib/ryal/web/models/payment_methods/credit_card.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Ryal.PaymentMethod.CreditCard do
+  @moduledoc """
+  Welcome to `Ryal.PaymentMethod.CreditCard`! Here, we store credit cards! OK,
+  not actually...
+
+  Well, we actually just store the information that would be of use for a user
+  and then the actual sensitive data with your payment gateways; the guys with
+  all that bank level security.
+
+  The `month` and `year` are both two digit fields that should come in as
+  strings. CVC is whichever you like it; it's a virtual string field. Finally,
+  that `number` field? It's a string as well. And don't worry about spaces,
+  tabs, or even line endings, we handle that :)
+  """
+
+  use Ecto.Schema
+
+  import Ecto.Changeset, only: [cast: 3, validate_required: 2, put_change: 3]
+
+  embedded_schema do
```

Comment:
Tokens should be part the model as well. Like the token id you'd get back from Stripe, Braintree, etc.